File: bro/src/main/java/me/xx2bab/bro/core/api/BinderApiFinder.java
package me.xx2bab.bro.core.api;

import me.xx2bab.bro.common.IBroApi;

public class BinderApiFinder implements IApiFinder {

    @Override
    public <T extends IBroApi> T getApi(Class<T> apiInterface) {
        return null;
    }

    @Override
    public IBroApi getApi(String nick) {
        return null;
    }

}

File: bro/src/main/java/me/xx2bab/bro/core/api/BinderApiFinder.kt
package me.xx2bab.bro.core.api

import me.xx2bab.bro.common.IBroApi

class BinderApiFinder : IApiFinder {
    override fun <T : IBroApi> getApi(apiInterface: Class<T>): T? {
        return null
    }

    override fun getApi(nick: String): IBroApi? {
        return null
    }
}